### ISSUE TITLE
Tokens, tokenizer, limits,, how far can it go?

### ISSUE THREAD

**Issue by zazoum-art** [kijai/ComfyUI-CogVideoXWrapper#226]
Hey @kijai  and anyone with the knowledge on this.
I'm training a local LLM that is **********UNCENSORED********* that AI assists SPECIFICALLY on vision and cogvideox prompt generation.

I MUST know why there is a limit in tokens, how the limit can be increased, it depends on a T5tokenizer (I don't know what this, is it separate model?is it a clip?) as I can see in the configs and there is a line that clearly sets the number of token. I'm NOT touching it, I'm not an expert on these, I don't want to ruin my setup that seems to work so far. I do things practically.

GOAL: Tell the local SPECIFIC LLM CogVideoX prompt generator, to translate my imagination not into a basic token size prompt, not to double or triple size, but a WHOLE ESSAY describing the motion of the output via the inputted image ( THAT IS: directing every single part of the input just as in real-life movie direction I have graduated from). WHY? The reason is to beat randomness on output. Randomness is commanding the machine. The words "commanding" and "directing" are different.

tldr: What's the possible number?
